style.css

```diff
@@ -360,6 +360,147 @@ h4 {
   color: gray;
 }
 
+.modal-overlay {
+  position: fixed;
+  background-color: #c1c7d0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  display: none;
+}
+
+.modal {
+  width: 90%;
+  height: 90%;
+  background: #fff;
+  margin-top: 25px;
+  border-radius: 20px;
+  position: relative;
+  padding-left: 15px;
+  display: none;
+  margin-left: 8%;
+  margin-right: 8%;
+  margin-bottom: 25px;
+  z-index: 99999;
+}
```

Comment:
[OPTIONAL] For the mobile version, the sections with classes `p.modal-description` and the `div.modal-language` should each appear as block elements in a vertical section and not horizontal as you have displayed. Kindly check and fix that.


Your readme             |  The readme template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-21 03-51-34](https://user-images.githubusercontent.com/65233860/197107841-70003fff-52a6-4e7d-9406-3dd77c03ae80.png)  |  ![Screenshot from 2022-10-21 03-50-50](https://user-images.githubusercontent.com/65233860/197107868-3b08b333-f6a2-4ba2-8384-f8cf42da6cc0.png)